lib/ProductOpener/Test.pm

```diff
@@ -743,6 +744,24 @@ sub normalize_products_for_test_comparison ($array_ref) {
 	return;
 }
 
+=head2 sort_products_for_test_comparison($array_ref, $sort_field)
+
+Sort products so that they are always in the same order
+
+=head3 Arguments
+
+=head4 array_ref
+
+Array of products
+
+=cut
+
+sub sort_products_for_test_comparison ($array_ref, $sort_field) {
```

Comment:
Good idea to make it a sub even if it's a one-liner, it really helps understand the code more easily.